spec.html

```diff
@@ -4020,6 +4045,184 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
+              The kind of the element.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Value]]
+            </td>
+            <td>
+              ~field~ and ~method~
+            </td>
+            <td>
+              any ECMAScript language value
+            </td>
+            <td>
+              The value of the field.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Get]]
+            </td>
+            <td>
+              ~accessor~
+            </td>
+            <td>
+              Function or Undefined
+            </td>
+            <td>
+              The getter for a private accessor.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Set]]
+            </td>
+            <td>
+              ~accessor~
+            </td>
+            <td>
+              Function or Undefined
+            </td>
+            <td>
+              The setter for a private accessor.
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-classfielddefinition-record-specification-type">
+      <h1>The ClassFieldDefinition Record Specification Type</h1>
+      <p>The ClassFieldDefinition type is a Record used in the specification of class fields.</p>
+      <p>Values of the ClassFieldDefinition type are Record values whose fields are defined by <emu-xref href="#table-classfielddefinition-fields"></emu-xref>. Such values are referred to as <dfn>ClassFieldDefinition Record</dfn>s.</p>
+      <emu-table id="table-classfielddefinition-fields" caption="ClassFieldDefinition Record Fields">
+        <table>
+          <tbody>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[Name]]
+            </td>
+            <td>
+              a Private Name, a String value, or a Symbol value.
+            </td>
+            <td>
+              The name of the field.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[Initializer]]
+            </td>
+            <td>
+              an ECMAScript function object, or ~empty~
+            </td>
+            <td>
+              The initializer of the field, if any.
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-private-names">
+      <h1>Private Names</h1>
+      <p>The <dfn>Private Name</dfn> specification type is used to describe a globally unique record which represents a private class element (field, method, or accessor). A Private Name may be installed on any ECMAScript object with PrivateFieldAdd or PrivateMethodAdd, and then read or written using PrivateGet and PrivateSet.</p>
```

Comment:
```suggestion
      <p>The <dfn>Private Name</dfn> specification type is used to describe a record which represents a private class element (field, method, or accessor). A Private Name may be installed on any ECMAScript object with PrivateFieldAdd or PrivateMethodAdd, and then read or written using PrivateGet and PrivateSet.</p>
```

Reply:
Tweaked this wording in https://github.com/tc39/ecma262/pull/1668/commits/214d6eae96bfa37b9eafc1a4f99241b6555ddaa7.